docs/web-apps/live-testing/live-cross-browser-testing.md

```diff
@@ -107,67 +107,6 @@ Know if your app is designed to run on internal or otherwise restricted networks
-| <img src={useBaseUrl('img/live-testing/download-vitals-icon.png')} alt="Download Vitals icon" width="40"/> | Download Vitals | Downloads the device vitals as a .txt file. |
-
-**Performance Metrics for Android/iOS Devices**
-
-The graph and csv file will contain these performance metrics for devices.
-
-| Metric | Description |
-| :--- | :--- |
-| `cpu_total` | System-wide CPU usage in percentage across all CPU cores. 4 cores at max use would be shown as a value of 400% |
-| `cpu_user` | CPU usage for user processes in percentage across all CPU cores. 4 cores at max use would be shown as a value of 400% |
-| `cpu_kernal` | Android OS CPU usage in percentage across all CPU cores. 4 cores at max use would be shown as a value of 400% |
-| `n_threads` | Total threads in use by the app |
-| `memory_size_kb` | Total threads in use by the app |
-| `memory_resident_kb` | Memory currently in use by app in kilobytes |
-| `memory_shared_kb` | Anonymous shared memory currently in use by system shared between app(s) and system |
-| `network_wifi_receive_b` | Data in bytes received over wifi connection |
-| `network_wifi_sent_b` | Data in bytes sent over wifi connection |
-| `network_mobile_receive_b` | Data in bytes received from the mobile carrier network |
-| `network_mobile_sent_b` | Data in bytes sent over mobile carrier network |
-
-:::note
-Device Vitals for live testing is currently in beta state, which means we are testing it on a few devices first. We will roll out to all devices in the coming weeks, please refer to this page for updates. See the full list of combinations available for beta below.
-:::
-
-<Tabs
-  defaultValue="iOS"
-  values={[
-    {label: 'iOS', value: 'iOS'},
-    {label: 'Android', value: 'Android'},
-  ]}>
-
-<TabItem value="iOS">
-
-**Web Tests (on Safari)**
-
-| Device/Platform | iPhone 11 | iPhone XR | iPhoneXS | iPhone X | iPhone 8 | iPhone 7 | iPhone 6 | iPhone 6 Plus | iPhone 6S Plus | iPhone 5S | iPhone SE | iPad Pro 11 2018 | iPad Pro | iPad Air 2019 | iPad 9.7 2017 | iPad 4 | iPad Mini 2 |
-| :--- | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
-| iOS 9.3.2 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
-| iOS 10.0.2 |  |  |  |  |  |  |  |  |  | X |  |  |  |  |  |  |  |
-| iOS 10.1 |  |  |  |  |  |  |  |  |  |  | X |  |  |  |  |  |  |
-| iOS 10.3.3 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  | X |  |
-| iOS 11.4 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
-| iOS 11.4.1 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
-| iOS 12.2 |  |  |  |  | X |  |  |  |  |  |  |  |  |  |  |  |  |
-| iOS 12.4.1 |  | X |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
-| iOS 13.0 |  |  | X |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
-| iOS 13.1 | X |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
-
-</TabItem>
-<TabItem value="Android">
```

Comment:
@kristofmuhi Just noticed the build for this PR was compiling with errors due to missing corresponding <TabItem>  and <Tabs> tags ~ to resolve this, the tabs for Android should be removed as well.

Reply:
@kristofmuhi I made the change. This is now resolved.